src/workerd/api/gpu/gpu-shader-module.c++

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include "gpu-shader-module.h"
+
+namespace workerd::api::gpu {
+
+jsg::Promise<jsg::Ref<GPUCompilationInfo>>
+GPUShaderModule::getCompilationInfo() {
```

Comment:
```suggestion
jsg::Promise<jsg::Ref<GPUCompilationInfo>> GPUShaderModule::getCompilationInfo() {
```

Reply:
Configured clang format for a 100 column limit.